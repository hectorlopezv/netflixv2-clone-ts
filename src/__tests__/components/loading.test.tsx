/* eslint-disable react/jsx-pascal-case */
import React from 'react';
import { render, fireEvent, getByTestId, getByAltText } from '@testing-library/react';


import Loading_, {Loading_ReleaseBody} from '../../components/Loading';


describe('Loading componen', () => {
    it('Renders Loading Compoennt <Loading />', () => {
        const {container, getByText, getByTestId} = render(
         < Loading_ src={"/images/karl.png"} data-testid="loading"/>
        );

        expect(getByTestId('loading')).toBeTruthy();
        expect(getByTestId('loading-picture')).toBeTruthy();
        expect(container.firstChild).toMatchSnapshot();
    });


    it('Renders the Loading <Loading_ReleaseBody />', () => {
        const {container, getByText, getByTestId} = render(
         < Loading_ReleaseBody/>
        );

        expect(container.firstChild).toMatchSnapshot();
    });

});