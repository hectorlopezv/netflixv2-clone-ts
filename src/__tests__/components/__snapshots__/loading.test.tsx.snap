// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Loading componen Renders Loading Compoennt <Loading /> 1`] = `
<div
  class="Loading__Spinner-sc-182vy79-0 cFKdut"
  data-testid="loading"
>
  <img
    class="Loading__Picture-sc-182vy79-1 kLuCXK"
    data-testid="loading-picture"
    src="/images/users//images/karl.png.png"
  />
</div>
`;

exports[`Loading componen Renders Loading Compoennt <Loading /> 2`] = `
<div
  class="Loading__Spinner-sc-182vy79-0 cFKdut"
  data-testid="loading"
>
  <img
    class="Loading__Picture-sc-182vy79-1 kLuCXK"
    data-testid="loading-picture"
    src="/images/users//images/karl.png.png"
  />
</div>
`;

exports[`Loading componen Renders the Loading <Loading_ReleaseBody /> 1`] = `null`;
